### DISCUSSION TITLE
How to restart a download

### DISCUSSION THREAD

**Discussion by TimHobson69** [aclap-dev/video-downloadhelper#1210] [Help and Q&A]
### Your question

My laptop shut itself down during a  download (Win 11, Google Chrome, myvidster.com). I can see the unfinished video in the VDH extension's list, but it is not doing anything (stuck at 21%). Is there a way to tell it to continue the download, or do I have to delete the file and start over? It's a big effing file, so I hate to have to start all over.

### Addon & CoApp version and details

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

VDH is not capable of restarting a failed download from the point of failure.  You need to reload the extension (that's a function in the VDH settings) & start the download over from the beginning.  We are all out of luck on this idea.

**Reply by TimHobson69** [reply]
Thanks. I was afraid that was the case. I appreciate the quick reply, and don't say you're "just another user." That's what these user groups are for!
